databricks-sdk-java/src/main/java/com/databricks/sdk/core/DatabricksConfig.java

```diff
@@ -463,7 +463,9 @@ public boolean isAzure() {
     if (host == null) {
       return false;
     }
-    return host.contains(".azuredatabricks.");
+    return host.contains(".azuredatabricks.net")
```

Comment:
Curious -- why do we need to limit it to `.net`?

Reply:
I'm just matching the Go SDK logic here.

Reply:
I see